back/engines/commercial/id_fake_sso/app/lib/id_fake_sso/fake_sso_verification.rb

```diff
@@ -24,8 +24,24 @@ def locked_attributes
       %i[first_name last_name]
     end
 
+    def other_attributes
+      %i[email]
```

Comment:
Do you think this could be derived from the `profile_to_user_attrs` method? I.e. these are the (non custom field) attributes returned by `profile_to_user_attrs`, above we define which of those are locked, so then the remaining ones have to be the 'other' ones?

Same for `other_custom_fields`?

Reply:
Could be, but might be easier to be explicit. I'll take a look